ingest-parsed-court-document-event-handler/src/main/scala/uk/gov/nationalarchives/ingestparsedcourtdocumenteventhandler/Lambda.scala

```diff
@@ -83,14 +83,13 @@ class Lambda extends LambdaRunner[SQSEvent, Unit, Config, Dependencies] {
             output.series,
             tdrUuid
           )
-          _ <- fileProcessor.createMetadataJson(metadata, metadataUri)
-          _ <- logWithFileRef(s"Copied metadata json to $outputBucket")
-          unusedFilesFromTre = fileNameToFileInfo.values
-            .filter(fi => !List(fileInfo, metadataFileInfo).contains(fi))
-            .map(_.id.toString)
-            .toList
-          _ <- IO.whenA(unusedFilesFromTre.nonEmpty) {
-            dependencies.s3.deleteObjects(outputBucket, unusedFilesFromTre) >> logWithFileRef("Deleted unused TRE objects from the root of S3")
+          _ <- fileProcessor.createMetadataJson(metadata, metadataPackage)
+          _ <- logWithFileRef(s"Copied metadata.json to bucket $outputBucket")
+          irrelevantFilesFromTre = fileNameToFileInfo.values.collect {
+            case fi if !List(fileInfo, metadataFileInfo).contains(fi) => fi.id.toString
+          }.toList
+          _ <- IO.whenA(irrelevantFilesFromTre.nonEmpty) {
+            dependencies.s3.deleteObjects(outputBucket, irrelevantFilesFromTre) >> logWithFileRef("Deleted unused TRE objects from the root of S3")
```

Comment:
Should this be "irrelevant" instead of "unused" here and in the test `"send a request to delete the unused tre file"`?